datastream-server/src/main/java/com/linkedin/datastream/server/CoordinatorConfig.java

```diff
@@ -59,6 +61,7 @@ public CoordinatorConfig(Properties config) {
     _defaultTransportProviderName = _properties.getString(CONFIG_DEFAULT_TRANSPORT_PROVIDER, "");
     _zkCleanUpOrphanConnectorTask = _properties.getBoolean(CONFIG_ZK_CLEANUP_ORPHAN_CONNECTOR_TASK, false);
     _zkCleanUpOrphanConnectorTaskLock = _properties.getBoolean(CONFIG_ZK_CLEANUP_ORPHAN_CONNECTOR_TASK_LOCK, false);
+    _performPreAssignmentCleanup = _properties.getBoolean(CONFIG_PERFORM_PRE_ASSIGNMENT_CLEANUP, true);
```

Comment:
Do you think we should turn on this feature by default without testing it in production?

Reply:
From testing, you mean we should just identify and print them?

Reply:
No, at least I meant, let's disable this by default, enable it through config when testing in cert, and once it looks good, add the config to enable it everywhere.